ui/admin/tests/unit/controllers/application-test.js

```diff
@@ -58,4 +60,60 @@ module('Unit | Controller | application', function (hooks) {
 
     assert.true(featuresService.isEnabled('ssh-target'));
   });
+
+  test('customRouteChangeValidator returns true when transitioning between different routes', function (assert) {
```

Comment:
Not blocking but I also wonder if these tests are similar enough to get parametrized?

Reply:
great suggestion!! 
updated the tests!